<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1116</width>
    <height>601</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>奇迹音乐播放器</string>
  </property>
  <property name="windowIcon">
   <iconset resource="player-assets.qrc">
    <normaloff>:/mplayer/mplayer.svg</normaloff>:/mplayer/mplayer.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,0">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="sideBar" native="true">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>200</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>220</width>
            <height>99999</height>
           </size>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="leftMargin">
            <number>20</number>
           </property>
           <property name="topMargin">
            <number>20</number>
           </property>
           <property name="rightMargin">
            <number>20</number>
           </property>
           <property name="bottomMargin">
            <number>20</number>
           </property>
           <item>
            <widget class="QWidget" name="title" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,6">
              <property name="sizeConstraint">
               <enum>QLayout::SetMinimumSize</enum>
              </property>
              <property name="leftMargin">
               <number>20</number>
              </property>
              <property name="topMargin">
               <number>20</number>
              </property>
              <property name="bottomMargin">
               <number>20</number>
              </property>
              <item>
               <widget class="QLabel" name="icon">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">image: url(:/mplayer/mplayer.png);</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>奇迹音乐</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="sideBarList">
             <property name="mouseTracking">
              <bool>true</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">#sideBarList{
background:transparent;
border:none;
}
#sideBarList::item:hover{
background:transparent;
}
#sideBarList::item:selected{
background:transparent;
}</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="mainContainer" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>800</width>
            <height>350</height>
           </size>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="topMargin">
            <number>20</number>
           </property>
           <property name="rightMargin">
            <number>20</number>
           </property>
           <item>
            <widget class="QWidget" name="horizontalWidget" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="mouseTracking">
              <bool>true</bool>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QWidget" name="windowMenuBar" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="mouseTracking">
                 <bool>false</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton{
    border-radius: 15;
}
QPushButton:hover{
	background-color: rgba(157, 157, 157, 100);
}
QPushButton:pressed{
	background-color: rgba(148, 148, 148, 120);
}</string>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <property name="spacing">
                  <number>5</number>
                 </property>
                 <item>
                  <widget class="QPushButton" name="min">
                   <property name="minimumSize">
                    <size>
                     <width>30</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="mouseTracking">
                    <bool>true</bool>
                   </property>
                   <property name="toolTip">
                    <string>最小化</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="player-assets.qrc">
                     <normaloff>:/mplayer/minus.svg</normaloff>:/mplayer/minus.svg</iconset>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="max">
                   <property name="minimumSize">
                    <size>
                     <width>30</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="mouseTracking">
                    <bool>true</bool>
                   </property>
                   <property name="toolTip">
                    <string>最大化</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="player-assets.qrc">
                     <normaloff>:/mplayer/square.svg</normaloff>:/mplayer/square.svg</iconset>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="close">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>30</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="mouseTracking">
                    <bool>true</bool>
                   </property>
                   <property name="toolTip">
                    <string>关闭</string>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">#close:hover{
	background-color: rgba(255, 0, 4, 180);
}
#close:pressed{
	background-color: rgba(197, 0, 3, 180);
}</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="player-assets.qrc">
                     <normaloff>:/mplayer/x.svg</normaloff>:/mplayer/x.svg</iconset>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QStackedWidget" name="content">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="mouseTracking">
              <bool>true</bool>
             </property>
             <property name="currentIndex">
              <number>3</number>
             </property>
             <widget class="AlbumView" name="album">
              <property name="mouseTracking">
               <bool>true</bool>
              </property>
             </widget>
             <widget class="FavView" name="fav">
              <property name="mouseTracking">
               <bool>true</bool>
              </property>
             </widget>
             <widget class="SearchForm" name="search">
              <property name="mouseTracking">
               <bool>true</bool>
              </property>
             </widget>
             <widget class="Settings" name="settings"/>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="PlayerControl" name="playerControl" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>80</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">#widget{
border-bottom-left-radius:26px;
border-bottom-right-radius:26px;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlayerControl</class>
   <extends>QWidget</extends>
   <header location="global">playercontrol.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AlbumView</class>
   <extends>QWidget</extends>
   <header location="global">albumview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SearchForm</class>
   <extends>QWidget</extends>
   <header location="global">searchform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FavView</class>
   <extends>QWidget</extends>
   <header location="global">favview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Settings</class>
   <extends>QWidget</extends>
   <header location="global">settings.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="player-assets.qrc"/>
 </resources>
 <connections/>
</ui>
